app/domain/logs.rb

```diff
@@ -351,42 +351,42 @@ module AuthnJwt
       )
 
       AddingJwtClaimToValidate = ::Util::TrackableLogMessageClass.new(
-        msg: "Adding JWT claim '{0-claim-name}' to list of mandatory claims to be validated",
+        msg: "Adding JWT claim, '{0-claim-name}' to list of mandatory claims to be validated...",
         code: "CONJ00069D"
       )
 
       CheckingJwtClaimToValidate = ::Util::TrackableLogMessageClass.new(
-        msg: "Checking if JWT claim '{0-claim-name}' is mandatory to validate",
+        msg: "Checking if JWT claimת '{0-claim-name}' is mandatory to validate...",
         code: "CONJ00070D"
       )
 
       FetchingJwtConfigurationValue = ::Util::TrackableLogMessageClass.new(
-        msg: "Fetching jwt signing key uri from '{0-resource-id}'",
+        msg: "Fetching JWT signing key URI from authenticator configuration'{0-resource-id}'...",
         code: "CONJ00071D"
       )
 
       FetchingJwksFromProvider = ::Util::TrackableLogMessageClass.new(
-        msg: "Fetching JWKS from '{0-uri}'",
+        msg: "Fetching JWKS from '{0-uri}'...",
         code: "CONJ00072D"
       )
 
       FetchJwtUriKeysSuccess = ::Util::TrackableLogMessageClass.new(
-        msg: "Fetched JWKS successfully",
+        msg: "Successfully fetched JWKS",
         code: "CONJ00073D"
       )
 
       ValidatingJwtSigningKeyConfiguration = ::Util::TrackableLogMessageClass.new(
-        msg: "Validating Jwt signing key uri configuration",
+        msg: "Validating JWT signing key URI configuration...",
```

Comment:
don;t understand what this. means? what is a signing key URI?

Reply:
Validating the one of this parameters and just one of this parameters is configured in the policy of the authenticator
`jwks-uri `and `provider-uri`

Reply:
@tzheleznyak  - Let's discuss....

Reply:
```suggestion
        msg: "Validating signing key URI configuration...",
```